build.sbt

```
@@ -255,6 +257,22 @@ lazy val testkit = project
       )
   )
 
+lazy val clustersharding = project
```

```suggestion
lazy val clusterSharding = project
```
To align with `cluster-sharding`.